### PR TITLE
adding reference to the wiki for the pi's address and credentials

### LINKED ISSUE #8: document how to ssh into raspberry pi
maybe just include ssh pi@(address..?) into the public docs

and say to ping you or me for the password?

just so that that we don't forget how to do it

### PR DESCRIPTION
**Pull request by itay-sho**
fixing #8, #10, #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +33,14 @@ a hardware & software project that lets people attending the [Recurse Center](ht
 
 be aware that everything runs without a virtualenv and with root permissions
 
+### Update / deploy new code
+1. ssh to the running rpi
+1. go to the cloned repo
+1. git pull
```
Comment (itay-sho) [inline README.md]:
```suggestion
1. `git pull`
```

File: README.md [hunk 2]
```diff
@@ -15,6 +15,10 @@ a hardware & software project that lets people attending the [Recurse Center](ht
 - the oauth/web/UI server is [here](https://github.com/gregsadetsky/checkintopus)
 - the hardware raspberry pi code is in this repo
 
+## Production environment
+We use a raspberry pi located in the hub's wifi
```
Comment (itay-sho) [inline README.md]: located is probably a better word for a physical location. rephrase it

**Review by itay-sho — COMMENTED** [review]
fixup commits

File: README.md (hunk 2 above)
Comment (gregsadetsky) [reply README.md]: chatgpt suggests "rhe raspberry pi connects to the hub's local wifi" which isn't fantastic

I do agree that "located" sounds like a physical location

actually, we should say where the pi is physically located as well :-) but yeah. just "the pi connects"  or "is connected" should be clear enough?

Comment (itay-sho) [reply README.md]: What do you think about the currently committed sentence?
`We use a raspberry pi that is located at the hub and connected to the network via wifi`

**Pull request merged**